tests/lowering/tensor_manipulation/test_permute.py

```diff
@@ -33,5 +33,34 @@ def test_permute(device, input_shapes):
     # Check the graph has be rewritten and contain ttnn ops
     nodes = list(option._out_fx_graphs[0].nodes)
     assert [node.target for node in nodes].count(ttnn.permute) == 1
+    # Check shape or result
+    assert result_before.size() == result_after.size()
+    # Check inference result
+    assert torch.allclose(result_before, result_after)
+
+
+# This tests tests the workaround for issue https://github.com/tenstorrent/tt-metal/issues/11191
+@pytest.mark.parametrize(
+    "input_shapes",
+    [
+        [(5, 1, 2)],
```

Comment:
@boris-drazic can you please help lookup if there are more shapes for the test in traces?

Reply:
Yes, there is a bunch.
These will pass with workaround:
```
Tensor<[1, 1024, 160]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 1024, 196]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 1024, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 1200, 320]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 128, 300]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 160, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 16384, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 16384, 32]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 19200, 64]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 256, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 256, 512]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 32, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 320, 300]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 4096, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 4096, 64]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 4800, 128]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 64, 256]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 64, 300]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 768, 1500]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 768, 196]> self = ?,<br>List[int] dims = [0, 2, 1]
```

These three still fail with `Page size must be divisible by sizeof(uint32_t) because buffers hold uint32_t values`:
```
Tensor<[1, 1024, 49]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 1280, 1369]> self = ?,<br>List[int] dims = [0, 2, 1]
Tensor<[1, 768, 49]> self = ?,<br>List[int] dims = [0, 2, 1]
```

I'll add the test cases to test.